ui/main/src/app/modules/share/card-detail/card-detail.component.html

```diff
@@ -9,5 +9,9 @@
 
 
 
-<link *ngFor="let cssUrl of hrefsOfCssLink" [href]="cssUrl" rel="stylesheet" type="text/css">
-<div id="opfab-card-template-detail" class="opfab-card-detail"  [innerHtml]="htmlContent" style="height: 100%;"></div>
\ No newline at end of file
+<of-spinner *ngIf="isLoading; else cardDetails"></of-spinner>
```

Comment:
i would prefer using hidden feature for card detail : 
```
<div id="opfab-card-template-detail" class="opfab-card-detail"  [innerHtml]="htmlContent" [hidden]="isLoading"  style="height: 100%;"></div>
<of-spinner *ngIf="isLoading"></of-spinner>
```
This is to avoid setting innerHtml before ng template is creating and thus having the div not existing. I do not know if it can arise , but using hidden guarantee that the <div> exists .